src/polytope/core/uuidgen.py

```diff
@@ -0,0 +1,56 @@
+import random
+
+# collision probability in 150,000 entries ~ 1%
+# lowercase alphabet + digit except [l, 1, o, 0]
+DEFAULT_ALPHABET = "abcdefghijkmnpqrstuvwxyz23456789"
+DEFAULT_LENGTH = 8
+
+
+class PolytopeUUID:
+    """UUID generator class."""
+
+    def __init__(self, alphabet: str = None, length: int = 0):
+        """! PolytopeUUID class initializer.
+
+        @param alphabet     alphabet for generating uuid
+        @param length       length of uuid
+        """
```

Comment:
Looks like we need some constraints on the parameters.
`alphabet` should be non-empty and distinct.
`length` should be positive.

If so, then setters must check them.

Reply:
Fixed it.

https://github.com/Studio-Polytope/Polytope/blob/f3d62c6a6c20de9c2e25fd69cc3c286ceee43ca5/src/polytope/utils/uuidgen.py#L21-L26